app/views/api/v2/models/main.json.rabl

```diff
@@ -3,3 +3,11 @@ object @model
 extends "api/v2/models/base"
 
 attributes :info, :created_at, :updated_at, :vendor_class, :hardware_model
+
+if params.has_key?(:include_permissions)
+  authorizer = Authorizer.new(User.current)
```

Comment:
This will still create a new authorizer (meaning, it will check the user's permissions again) for every object on the list. in the regular controllers (`ApplicationController`), we have a helper method `authorizer` that creates one authorizer per request that is used to check all of the permissions. Perhaps move that method to the authorizer helper as well?